Sources/AppcuesKit/Presentation/Debugger/FloatingView/DebugViewController.swift

```diff
@@ -16,12 +17,14 @@ internal class DebugViewController: UIViewController {
         set { debugView.delegate = newValue }
     }
 
-    private var debugView = DebugView()
+    private lazy var debugView = DebugView(mode: mode)
 
-    private let panelViewController: UIViewController
+    let mode: DebugMode
+    private let viewModel: DebugViewModel
 
-    init(wrapping panelViewController: UIViewController) {
-        self.panelViewController = panelViewController
+    init(viewModel: DebugViewModel, mode: DebugMode) {
```

Comment:
Does this handle the case where the debugger is loaded in one mode and then the other is tried? Like if I have the regular debugger open and then try the capture deep link? Everything looks tied to the init as far as I can see.

Reply:
That part is currently handled a level up, in `UIDebugger.show(mode: DebugMode)` -- it will hide the old debugger instance and create a new one entirely when switching modes.

Reply:
Gotcha. I guess it makes sense to just nuke it and start over since we want the floating button, etc to get reset when launching the new mode.